src/core/pages/signin.tsx

```diff
@@ -59,16 +72,17 @@ export default function SigninPage(props: SignInServerPageParams) {
 
   return (
     <div className="signin">
-      
-      { theme.brandColor && <style
```

Comment:
Not 100% sure if this is caused by a wrong Prettier config on my end or it's still a valid change.

Let me know if I missed something here, thanks! 👀